static/js/markdown.js

```diff
@@ -72,6 +72,18 @@ exports.apply_markdown = function (message) {
             }
             return undefined;
         },
+        groupMentionHandler: function (name) {
+            var group = user_groups.get_user_group_from_name(name);
+            if (group !== undefined) {
+                if (user_groups.is_member_of(group.id, people.my_current_user_id())) {
+                    push_uniquely(message.flags, 'mentioned');
+                }
+                return '<span class="group-mention" data-user-group-id="' + group.id + '">' +
```

Comment:
The class here should have been `user-group-mention`, so this didn't work.  I fixed that before merging

Reply:
Oops.